src/ChatPage.cpp

```diff
@@ -758,9 +758,9 @@ ChatPage::showUnreadMessageNotification(int count)
 
         // TODO: Make the default title a const.
         if (count == 0)
-                emit changeWindowTitle("nheko");
+                emit changeWindowTitle(0);
         else
-                emit changeWindowTitle(QString("nheko (%1)").arg(count));
+                emit changeWindowTitle(count);
```

Comment:
I think you can delete this and connect to `unreadMessages` signal directly instead.